src/geojson.rs

```diff
@@ -97,6 +97,12 @@ impl From<FeatureCollection> for GeoJson {
     }
 }
 
+impl From<Vec<Feature>> for GeoJson {
+    fn from(features: Vec<Feature>) -> GeoJson {
+        GeoJson::from(features.into_iter().collect::<FeatureCollection>())
```

Comment:
There are other ways to write this that avoid the collect. I haven't benchmarked. Any sense if this one is OK or if it's worth doing a more thorough comparison?